FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuthErrors.h

```diff
@@ -429,6 +429,11 @@ typedef NS_ERROR_ENUM(FIRAuthErrorDomain, FIRAuthErrorCode){
         describing which step of the JWT parsing process failed.
      */
     FIRAuthErrorCodeMalformedJWT = 18000,
+
+    /** Raised when an Cloud Function returns a blocking error. Will include a message returned from
+     * the function.
+     */
+    FIRAuthErrorCodeBlockingCloudFunctionError = 18001,
```

Comment:
Please change it to 17105.

Reply:
Done!

Reply:
Sorry, forgot to mention that please move this section to L415.